src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,35 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+
+  grid-template-columns: repeat(4, $card-width);
+
+  gap: 48px;
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, $card-width);
+  }
+
+  &__card {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $card-width;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+
+  }
+
+  &__card:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+}
```

Comment:
You've already set the gap property for catalog, I think, it makes the margins for the cards redundant:broom: